### PR TITLE
Add a bio file for Ondine

### PR DESCRIPTION
**Pull request by ondine**
This Pull Request adds a bio file for @ondine as requested in #5.

@githubteacher I would :heart: if you would put some :eyes: on the bio and share any :thought_balloon: you have.

### CODE REVIEW

**Comment by stevensgill** [comment]
Your bio looks good to me!

File: docs/ondine.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Ondine Bio
+
+- **Name:** Ondine
+- **Occupation:** Technical Writer
+- **Location:** Emeryville
```
Comment (stevensgill) [inline docs/ondine.md]: Which state is Emeryville in?

**Comment by ondine** [comment]
Fixes #5

**Comment by ondine** [comment]
Fixes #5

**Pull request closed without merging**